frontend/src/components/crop/SelectionOperations.ts

```diff
@@ -6,15 +6,19 @@ export type SelectionOperations = {
   onScreenBeingModified: (selection: SelectionBase) => boolean;
   forceRefresh: () => void;
   removeAll: () => void;
+  toPrevious: () => void;
+  undo_possible: boolean;
```

Comment:
Prefer camel case to snake case:
```suggestion
  undoPossible: boolean;
```